requestor/run_service_manager.py

```diff
@@ -0,0 +1,44 @@
+import asyncio
+
+from service_manager import ServiceManager
+import erigon_services
+
+
+def state(service_wrapper):
+    if not service_wrapper.started:
+        status = 'pending'
+    elif service_wrapper.stopped:
+        status = 'stopped'
+    else:
+        #   NOTE: this is erigon-specific thing, we don't use state of instance
+        #   beacues from our POV it is "running" when we know the url
+        if service_wrapper.service.url is None:
```

Comment:
If you used multiple `return`s as I've suggested in the previous comment this would have to go in an `else` block at all. :stuck_out_tongue: You could just separate it with a black line and avoid nesting.